ci/scripts/go_build.sh

```diff
@@ -42,6 +42,13 @@ main() {
         make all
 
         popd
+
+        mkdir -p "${install_dir}/lib"
+        if [[ $(go env GOOS) == "linux" ]]; then
+            cp ./pkg/libadbc_driver_flightsql_go.so "${install_dir}/lib"
+        else
+            cp ./pkg/libadbc_driver_flightsql_go.dylib "${install_dir}/lib"
```

Comment:
at @kou's suggestion I've removed the `_go` from the shared lib name in the cmake PR, do you wanna do the same here? or should i just do that when i integrate these into my cmake changes?

Reply:
I removed them here.